Cloud Application Manager/Automating Deployments/using-kubernetes.md

```diff
@@ -79,11 +79,12 @@ In YAML it is valid to define multiple documents in a single file, separated by
 apiVersion: v1
 kind: Service
 metadata:
-  name: frontend
+  name: frontend-\{{ service }}
+  namespace: \{{ namespace }}
 ...
 ```
 
-Kubernetes templates shall not include a *namespace* property in their metadata. Since the namespace is determined and forced by the Deployment Policy Box used to deploy the Instance, the deployment will fail if the template requests a namespace different from the one set with the Policy.
+Kubernetes templates shall not include a hard-coded *namespace* property in their metadata. Since the namespace is determined and forced by the Deployment Policy Box used to deploy the Instance, the deployment will fail if the template requests a namespace different from the one set with the Policy. The namespace set in the Deployment Policy can be referred with the  `namespace` variable.
```

Comment:
```suggestion
Kubernetes templates shall not include a hard-coded *namespace* property in their metadata. Since the namespace is determined and forced by the Deployment Policy Box used to deploy the Instance, the deployment will fail if the template requests a namespace different from the one set with the Policy. The namespace set in the Deployment Policy can be referred with the `namespace` variable.
```